Dockerfile

```
@@ -1,4 +1,9 @@
 FROM gcr.io/distroless/static@sha256:d9f9472a8f4541368192d714a995eb1a99bab1f7071fc8bde261d7eda3b667d8
+
+# Add metadata
+LABEL maintainer="External Secrets Operator" \
```

it is better that this is our cncf maintainer email:

```suggestion
LABEL maintainer="cncf-externalsecretsop-maintainers@lists.cncf.io" \
```